pkg/internalapis/cnsvolumeinfo/cnsvolumeinfoservice.go

```diff
@@ -159,10 +159,10 @@ func (volumeInfo *volumeInfo) CreateVolumeInfo(ctx context.Context, volumeID str
 	err := volumeInfo.k8sClient.Create(ctx, &cnsvolumeinfo)
 	if err != nil {
 		if !apierrors.IsAlreadyExists(err) {
-			log.Errorf("failed to create CR for cnsvolumeInfo CR. Error: %v", err)
-			return err
+			return logger.LogNewErrorf(log, "failed to create CR for cnsvolumeInfo %v in the namespace: %q. "+
+				"Error: %v", cnsvolumeinfo, err, csiNamespace)
```

Comment:
I guess arguments are jumbled. Should be `csiNamespace, err`

Reply:
fixed.